### PR TITLE
 solved level1-hello typescript 

### PR DESCRIPTION
**Pull request by jes-alpha**

### CODE REVIEW

**Comment by alessiopcc** [comment]
Remember to use as base branch for pull request the branch you want to merge into. For this repository is the corresponding level branch, not master.

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export class Challenge {
+    num : number;
+    constructor(){
+    }
+    public buzz(i:number){
+        return new Promise(function(resolve, reject){
+            if(i%3==0 && i%5==0)
+                resolve("fizzbuzz");
```
Comment (alessiopcc) [inline src/index.ts]: This is ok.
Just a consideration: remember that `resolve` and `reject` are simple callback and they don't stop the flow, any line after that will be executed. If you want to break use that in combination with `return` (i.e. `return resolve();`)

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+            if(i%3==0)
+                resolve("fizz");
+            if(i%5==0)
+                resolve("buzz");
+            else
```
Comment (alessiopcc) [inline src/index.ts]: In this case (without `else if`) this refers only to last `if`. Maybe this wasn't what you had in mind ;)

File: src/index.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export class Challenge {
+    num : number;
```
Comment (alessiopcc) [inline src/index.ts]: This is unused, remove it.
For the future: remember to mark members always as `public`, `private` or `protected`. For convention non-public members have a leading underscore. 
For example: `public num: number;` and `private _num: number;`

File: src/index.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    constructor(){
```
Comment (alessiopcc) [inline src/index.ts]: Empty constructors are the default and can be avoided.
Also try to always mark it with its visibility just for better clarity (`public` is the default level and is good in most cases)

**Comment by alessiopcc** [comment]
Started the review before the last commit, sorry. Style is good now, good job.

File: src/index.ts [hunk 5]
```diff
@@ -0,0 +1,17 @@
+export class Challenge
+{
+    public buzz(i: number)
```
Comment (marmos91) [inline src/index.ts]: Generally we prefer to name function parameters with more intuitive name. So please avoid `n`, `i`, `c` and prefer `entry`, `count`, `peers`, etc...

**Pull request closed without merging**